src/test/java/com/qa/restassured/PracticeWithLogs.java

```diff
@@ -0,0 +1,122 @@
+package com.qa.restassured;
+
+import io.restassured.config.LogConfig;
+import io.restassured.http.Header;
+import io.restassured.http.Headers;
+import io.restassured.response.Response;
+import org.testng.annotations.Test;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Set;
+
+import static io.restassured.RestAssured.config;
+import static io.restassured.RestAssured.given;
+
+public class PracticeWithLogs {
+// @Test
+//public void verifyResponseWithLogs() {
+//        given()
+//        .baseUri("https://restful-booker.herokuapp.com")
+//        .header("x-api-key", "PMAK-65a61915fb1cc57ce33451e6-f381c9710bd9ad1950681626aec7a728e7")
+//        .log().headers()
+//        .when()
+//        .get("/booking/2019")
+//        .then()
+//        .log().body()
+//        .assertThat()
+//        .statusCode(200);
+//}
+//        @Test
+//        public void verifyWithIfErrorLogs() {
+//        given()
```

Comment:
Any reason why these codes have been commented?
A suggestion: 

1. Never commit commented code 
2. If you really need some code to be commented for somereason, please add a comment statement explaining with reason
Example:

// Below code is commented temporarily due to a bug and dependency problem, this should be uncommented post fix
SOME COMMENTED CODE HERE...

Reply:
while executing all test cases are executed so I was commented all and run a single to see the proper output. 

Okay I will tack care further to provide comment statement while commit my code.

Reply:
@patrabimal That is a wrong approach. Please do not follow that practice. 

- Imagine you are working with a team and you commented multiple tests and pushed to repo
- Other team members when they get latest pull , they will also find these tests commented
- Most importantly, your commented code will be part of your pull request and it will not be approved to be merged to main because that will stop them running as part of regression

What is another approach to your problem?

1. You can use tag or group from testNG to run only specific set of tests
2. You can run specific tests via IDE
3. Only after you are done with new test do a full run via maven command line and then do commit